app/migration/mappers/programme_type_mapper.rb

```diff
@@ -0,0 +1,19 @@
+module Mappers
+  class ProgrammeTypeMapper
+    # Maps the ECF1 values to the ECF2 values
+    MAPPING = {
+      "core_induction_programme" => "school_led",
+      "design_our_own" => "school_led",
+      "school_funded_fip" => "provider_led",
+      "full_induction_programme" => "provider_led"
+    }.freeze
+
+    def initialize(value)
+      @value = value
+    end
+
+    def mapped_value
+      MAPPING[@value]
```

Comment:
I wonder if this should `raise` if there is no mapping?  What do you think?

Reply:
I checked the values in the database and they look all they have one of the keys in the MAPPING. Don't expect any nil values.